environment

```diff
@@ -333,19 +333,33 @@ _hibernate() {
 		make -C lambdas && {
 			cd terraform &&
 			make validate
-			terraform destroy -target aws_elasticsearch_domain.index && {
+			terraform destroy \
+			    -target aws_cloudwatch_metric_alarm.indexercachehealth \
+			    -target aws_cloudwatch_metric_alarm.servicecachehealth \
+			    -target aws_elasticsearch_domain.index \
```

Comment:
This file exclusively uses tabs for indentation. These three lines use a mix of spaces and tabs.